### DISCUSSION TITLE
Getting 0 Content Field Required after sending an image

### DISCUSSION THREAD

**Discussion by j-taptok** [open-webui/open-webui#1883] [Q&A]
I am getting this error after sending an image using Opus.

AnthropicException - AnthropicException - {"type":"error","error":{"type":"invalid_request_error","message":"messages.0.content.0.text.text: Field required"}}

Unless I remove the prompt with the image it gives that error. Does anyone know how to avoid this?

<img width="902" alt="image" src="https://github.com/open-webui/open-webui/assets/98992496/2df4b737-13b1-4314-a670-4e696393e35a">

**Comment by cheahjs** [comment]
It looks like LiteLLM does not flag the Claude 3 family as vision capable, and does not translate the OpenAI image format into Anthropic's, hence why Anthropic is complaining that there's a missing text field.

**Reply by j-taptok** [reply]
@cheahjs that might be the case.

**Comment by spammenotinoz** [comment]
Appears fixed in the latest version of litellm

**Comment by JuanjoJG-WATA** [comment]
It is indeed fixed. For me at least.

**Reply by spammenotinoz** [reply]
Providing the images are small 300k you won't have an issue, it errors out at 5MB (current limit supported by Anthropic).
